src/index.tsx

```diff
@@ -150,24 +178,7 @@ async function activatePlugin(
 ): Promise<void> {
```

Comment:
Because you've removed the only blocking `await` statement, you can actually just edit the definition `activatePlugin` to be entirely synchronous. Change its definition to:

```ts
function activatePlugin(
  app: JupyterFrontEnd,
  browserFactory: IFileBrowserFactory,
  notebookTracker: INotebookTracker,
  translator: ITranslator,
  restorer: ILayoutRestorer,
  advancedOptions: Scheduler.IAdvancedOptions,
  telemetryHandler: Scheduler.TelemetryHandler,
  launcher: ILauncher | null
): void {
  ...
}
```

Reply:
That's a really good point. Done.